vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -170,19 +170,25 @@ class MessageActionsViewModel @AssistedInject constructor(@Assisted
     private fun computeMessageBody(timelineEvent: TimelineEvent): CharSequence {
         return when (timelineEvent.root.getClearType()) {
             EventType.MESSAGE,
+            EventType.ENCRYPTED,
             EventType.STICKER     -> {
-                val messageContent: MessageContent? = timelineEvent.getLastMessageContent()
-                if (messageContent is MessageTextContent && messageContent.format == MessageFormat.FORMAT_MATRIX_HTML) {
-                    val html = messageContent.formattedBody
-                            ?.takeIf { it.isNotBlank() }
-                            ?.let { htmlCompressor.compress(it) }
-                            ?: messageContent.body
-
-                    eventHtmlRenderer.get().render(html)
-                } else if (messageContent is MessageVerificationRequestContent) {
-                    stringProvider.getString(R.string.verification_request)
-                } else {
-                    messageContent?.body
+                when (timelineEvent.root.isRedacted()) {
```

Comment:
As a remark, I feel strange to test boolean in when like this.
I would have written:
```
when {
    timelineEvent.root.isRedacted() -> /**/
    else -> /**/
}
```